### PR TITLE
Frontend Prototype 1

### PR DESCRIPTION
**Pull request by colinmegill**
1. 

image

2. 

image

3. 

image

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
One more question. In the `cells` field for the pancreas dataset, the value is `cells`. Is that because the data is missing from the back-end?

File: dcp_prototype/frontend/src/components/layout.css [hunk 1]
```diff
@@ -14,19 +14,19 @@ div {
 }
 
 div[role="group"][tabindex] {
-  background-color: white;
+  background-color: white !important;
```
Comment (ambrosejcarr) [inline dcp_prototype/frontend/src/components/layout.css]: Could you add a comment here to flag that this is to override global css being inserted by Gatsby? If it were me I'd also either drop a #todo here to file a ticket to address this later.

File: dcp_prototype/frontend/gatsby-browser.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
```
Comment (Bento007) [inline dcp_prototype/frontend/gatsby-browser.js]: Remove this file

File: dcp_prototype/frontend/gatsby-node.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+/**
```
Comment (Bento007) [inline dcp_prototype/frontend/gatsby-node.js]: Remove this file

File: dcp_prototype/frontend/gatsby-ssr.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/**
```
Comment (Bento007) [inline dcp_prototype/frontend/gatsby-ssr.js]: Remove this file

**Comment by Bento007** [comment]
The instructions in the readme should be sufficient to generate any of the needed gatsby files.

File: dcp_prototype/frontend/package.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    "develop": "gatsby develop",
+    "format": "prettier --write \"**/*.{js,jsx,json,md}\"",
+    "start": "npm run develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gatsbyjs/gatsby-starter-default"
```
Comment (bkmartinjr) [inline dcp_prototype/frontend/package.json]: wrong url?

File: dcp_prototype/frontend/package.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
```
Comment (bkmartinjr) [inline dcp_prototype/frontend/package.json]: wrong url?

**Comment by liaprins-czi** [comment]
@colinmegill Not blocking merge, but most of the text should be Monstserrat, only the major headings (to replace DIN) should be in the current typeface, just FYI.

**Pull request merged**